src/PhaseState.jl

```diff
@@ -169,72 +173,96 @@ Maintains diffusion limitations if the phase has diffusionlimited=true
 end
 export getkfkrev
 
-@inline function getkfkrevs(phase::U,T::W1,P::W2,C::W3,N::W4,ns::Q1,Gs::Q2,diffs::Q3,V::W5;kfs::W6=nothing) where {U<:AbstractPhase,W6,W5<:Real,W1<:Real,W2<:Real,W3<:Real,W4<:Real, Q1<:AbstractArray,Q2<:Array{Float64,1},Q3<:AbstractArray}
+@inline function getkfkrevs(phase::U,T::W1,P::W2,C::W3,N::W4,ns::Q1,Gs::Q2,diffs::Q3,V::W5,phi::W7;kfs::W6=nothing) where {U<:AbstractPhase,W7,W6,W5<:Real,W1<:Real,W2<:Real,W3<:Real,W4<:Real, Q1<:AbstractArray,Q2<:Array{Float64,1},Q3<:AbstractArray}
```

Comment:
So I thought about this when I was writing it. Function dispatch can be used, but it will double the number of getkfkrevs functions. I haven't found a good reason to do so though, the overhead compared to the krevs calculation should be very negligible and I don't anticipate it will affect what objects we can send through, but if it does later it's easy to add the other dispatch functions when needed.